timesheets/Code.gs

```diff
@@ -0,0 +1,148 @@
+      if (notifiedStatus == 'NOTIFIED') {
+        continue;
+      }
+           
+      var range = sheet.getRange(i, 3);
+      var email = range.getValue();
+      var dropdownValue = sheet.getRange(i,APPROVAL_COLUMN);    
+      
+      // approval email
+      if (dropdownValue.getValue() == 'APPROVED') {
+        var message = 'Your timesheet has been approved';
+        var subject = 'TimeSheet Approval';
+      } else if (dropdownValue.getValue() == 'NOT APPROVED') {
+        var message = 'NOT APPROVED';
+        var subject = 'TimeSheet not Approved';
+      } else if (dropdownValue.getValue() == 'IN PROGRESS') {
+        continue;
+      } 
+      
+      MailApp.sendEmail(email, subject, message);
+      var notifiedUpdate = sheet.getRange(i, lastCol).setValue('NOTIFIED');
+    }
+}
+
+
+/**
+* Runs the "Add Approval Column" script, adding an approval column at the end of the sheet, 
+* made entirely of checkboxes.
+* Adds "NOTIFIED STATUS" column indicating whether or not an employee has yet been e mailed.
+*/
+function addApprovalColumn() { 
+  // adds new column
+  var sheet = SpreadsheetApp.getActiveSheet();
+  var lastCol = sheet.getLastColumn();
+  var lastRow = sheet.getLastRow();
+  sheet.insertColumnAfter(lastCol);
+  
+  // sets new column title
+  var cell = sheet.getRange(1, lastCol+1);
+  cell.setBackground('ffe3ee').setValue('APPROVAL');
+  
+  // make sure approval column is all drop-down menu
+  for (i = 2; i <= lastRow; i++) {
+    var dropdownBox = sheet.getRange(i,lastCol+1);
+    var dropdownValues = ['APPROVED', 'NOT APPROVED', 'IN PROGRESS'];
+    var rule = SpreadsheetApp.newDataValidation().requireValueInList(dropdownValues).build();
+    dropdownBox.setDataValidation(rule);
+    dropdownBox.setValue('IN PROGRESS');
+  }
+  
+  // add the notified column
+  sheet.insertColumnAfter(APPROVAL_COLUMN); //global
+  var cell = sheet.getRange(1, APPROVAL_COLUMN + 1);
+  cell.setBackground('ffe3ee').setValue('NOTIFIED STATUS'); 
+  
+  // sets notified status to NOT NOTIFIED
+  for (var i = 2; i <= lastRow; i++) {
+    var notifiedCell = sheet.getRange(i, APPROVAL_COLUMN + 1);
+    var notifiedValues = ['NOTIFIED', 'NOT NOTIFIED'];
+    var rule = SpreadsheetApp.newDataValidation().requireValueInList(notifiedValues).build();
+    notifiedCell.setDataValidation(rule);
+    notifiedCell.setValue('NOT NOTIFIED');
+  }
+}
+
+/**
+* Runs the "Calculate Pay" script, creating a new column with each employee's 
+* pay based on their reported hours and hourly wage.
+*/
+function calculatePay() {
+  // adds new column
+  var sheet = SpreadsheetApp.getActiveSheet();
+  var lastCol = sheet.getLastColumn();
+  var lastRow = sheet.getLastRow();
+  sheet.insertColumnAfter(lastCol);
+  
+  // sets new column title
+  var cell = sheet.getRange(1, lastCol+1);
+  cell.setValue('WEEKLY PAY');
+  
+  // goes through every employee
+  for (var i = 2; i <= lastRow; i++) {
+    var totalHours = 0;
+    var numHrs = 0;
```

Comment:
Avoid using abbreviations like `Hrs`, as it can be hard to read for non-native speakers. I think `num` is fine, as it's a bit more standard.

Reply:
Sounds great - changed this for all of the variables.